src/test/applied_science/js_interop_test.cljs

```diff
@@ -499,4 +499,15 @@
         (is (= (j/get o .-yyyyyy) (if advanced? nil 10)))
         (is (= (j/get o :yyyyyy) 10))
         (is (= (j/get o .-zzzzzz) (if advanced? nil 20)))
-        (is (= (j/get o :zzzzzz) 20))))))
+        (is (= (j/get o :zzzzzz) 20)))))
+
+  (testing "object merging"
```

Comment:
Can move these to be included in the `are` test beginning on L24, and include `apply` variants so that both the macro and function are tested. (This wouldn't work for the first test, as it is a `not=`, so that can either stay here or be removed, as we don't check anywhere else that these operations "don't work" for clj maps, I think that can be assumed)